nucypher/cli/painting/staking.py

```diff
@@ -34,7 +34,7 @@
 )
 from nucypher.cli.painting.transactions import paint_receipt_summary
 
-STAKE_TABLE_COLUMNS = ('Idx', 'Value', 'Remaining', 'Enactment', 'Termination', 'Status')
+STAKE_TABLE_COLUMNS = ('Slot', 'Value', 'Remaining', 'Enactment', 'Termination', 'Boost', 'Status')
```

Comment:
```suggestion
STAKE_TABLE_COLUMNS = ('Slot', 'Value', 'Remaining', 'Enactment', 'Termination', 'Long Lock Boost', 'Status')
```
This might make it even clearer

Reply:
> This might make it even clearer

Meh, I think this will make the table header too large to render on most terminals. Personally, I prefer to stick with the one word table convention here.  If more information is needed it does not need to be presented here.